s2i/bin/run

```diff
@@ -443,10 +444,16 @@ else
 						GIT_TOKEN="${GIT_TOKEN}"
 						FILES="../${whlff} ../build_info.yaml"
 						ls -l
-						if [ "$USE_PAGURE_COMMIT" == "1" ]; then
+						if [[ "$USE_PAGURE_COMMIT" == "1" ]]; then
 							git clone $PAGURE_COMMIT_REPO
 							cd tensorflow
-							source ../pagure_commit.sh "${NOTES}" "${FILES}" "${whlff}"
+							source ../pagure_commit.sh "${TENSORFLOW_BUILD_DIR_NAME}" "${NOTES}" "${FILES}" "${whlff}"
+						elif [[ "$USE_BOTH" == "1" ]]; then
+							(git clone $PAGURE_COMMIT_REPO;
+							cd tensorflow;
+							source ../pagure_commit.sh "${TENSORFLOW_BUILD_DIR_NAME}" "${NOTES}" "${FILES}" "${whlff}";) & (git clone $GIT_RELEASE_REPO;
```

Comment:
When running in parallel should we have error handling if pagure commit fails ?
 Will the second command work if first command fails? I guess yes.But  want to confirm.

Reply:
```
$ echod "hello" & echo "world"
[1] 28854
world
-bash: echod: command not found
[1]+  Exit 127                echod "hello"
$ echo $?
0

$ echo "hello" & echod "world"
[1] 28861
hello
-bash: echod: command not found
[1]+  Done                    echo "hello"
$ echo $?
127



$ echod "hello" & echod "world"
[1] 28858
-bash: echod: command not found
-bash: echod: command not found
[1]+  Exit 127                echod "hello"
$ echo $?
127
```

Looks like running in parallel may be a bad idea.Because it may say the command is success or failure depending upon the last process status.
lets run sequentially and if the pagure commit fails fail the run.Because pagure is more important that github.

Reply:
needs code change imo.